### PR TITLE
Update changelog to 3.0.3/3.1.0

### LINKED ISSUE #46: Changelog 3 years old
Newest entry is v1.6.x back from 2016 while, at least referring to _F-Droid_, there is jump in version more than a full generation up to v2.2 for real. Please (heavily) consider to keep the Changelog up-to-date.

### PR DESCRIPTION
**Pull request by derEisbaerX**
Fixes #46 to a large extent if you also merge #89

### CODE REVIEW

**Comment by derEisbaerX** [comment]
Dear maintainers,
could you please review and eventually merge my pull requests and release a new version afterwards?
I propose version 3.1.0 because there are huge, but no structural changes in the app.

**Review by Kamuno — COMMENTED** [review]
I'll leave this open and merge it as soon as the other PRs are merged :)

**Comment by Kamuno** [comment]
Since this PR is very fitting, I just want to say thank you for your work so far 👍

**Review by Kamuno — APPROVED** [review]

**Pull request merged**
